packages/terra-application-navigation/src/extensions/Extensions.module.scss

```diff
@@ -0,0 +1,8 @@
+/* stylelint-disable selector-class-pattern */
```

Comment:
Does this need to be disabled for this file?

Reply:
Updated.